resources/code_examples/analytics/complete_example.yml

```
@@ -17,7 +17,7 @@
       revenue:       '4417.58',               // Grand Total. Includes Tax and Shipping. Does not include payment costs.
       shipping:      '5.45',                  // Total Shipping Cost. Does not include payment costs.
       tax:           '1014.79',               // Total Tax.
-      paid_by:       'example_paid_by',       // [Optional] Payment method type, e.g. bank_transfer.
+      paid_by:       'example_paid_by',       // Payment method type, e.g. bank_transfer. Required for Technology categories.
       paid_by_descr: 'Example paid_by_descr'  // [Optional] Payment method description, e.g. Bank transfer.
```

Based on the task description we must include the same comment for `paid_by_descr`.

---

agreed, there are way too many examples of shops that report something very generic in `paid_by` such as `xpaymentX` or `alg_custom_gateway_X`, which by itself is not indicative at all

---

we can also consider changing this example similarly to my other suggestion:

```suggestion
      paid_by:       'card',                  // Payment method type.
      paid_by_descr: 'Πληρωμή με κάρτα'       // Payment method description as free text.
```